contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,286 @@
+
+At PostHog, we define activation metrics as a set of actions where a company is much more likely to be using the product 3 months after starting. For [session replay](/session-replay), after analyzing multiple potential metrics, we found that watching 5 or more replays and setting a replay filter at least once were the actions that resulted in the highest retention rate.
+
+![Session replay activation metric](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_06_at_10_40_35_2x_a07c5ba4fd.png)
+
+Actions that don't make it into your activation metric can still be good, but they don't help you predict retention. For example, we want users to invite teammates, but we found that doing so doesn't mean they stick around, so optimizing this isn't the best use of our time.
+
+![Product analytics activation](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_13_at_09_53_01_2x_2c8bc81896.png)
+
+## How we found our activation metric
+
+How did we settle on watching 5 or more replays and setting a replay filter at least once? We tested a lot of different metrics by:
+
+1. Creating a big list of events that we thought might "hook" users into using our product.
+
+2. Creating groups of 3-5 events to test together. We aimed for 5-10 different groups, including the number of times the event occurs (e.g. watched 5 replays vs. watched 1 replay). Sometimes users have to do something more than once to reach their "ah-ha" moment.
+
+![Event groups](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_06_at_11_36_20_2x_89a95ebdf3.png)
+
+3. Writing a query to test how the event groups correlate with retention (see below).
+
+4. Comparing each event group against our average retention rate for all users. We looked for a retention percentage that's nominally higher than our average retention rate.
+
+After testing multiple event groups, we found that watching 5 or more replays and setting a replay filter at least once was the set of events that resulted in the highest retention rate, meaning it was our activation metric.
+
+## Our query for testing potential activation metrics
+
+How can you recreate this process for yourself? Creating the groups of potential activation events is easy, but the retention testing query is the tricky part. 
+
+To help you out, we've included the exact query we used below. If you use PostHog, you can create a [new SQL insight](https://us.posthog.com/insights/new) and use our same exact query to explore your data and find your activation metric. First, I'll break the query into parts to explain what's happening, then I'll provide the whole thing in one chunk [at the end](#the-final-combined-query).
```

Comment:
This might read a bit nicer.
```suggestion
To help you out, we've included the exact query we used below. First, I'll break the query into parts to explain what's happening, then I'll provide the whole thing in one chunk [at the end](#the-final-combined-query).

If you use PostHog, you can create a [new SQL insight](https://us.posthog.com/insights/new) and use our same exact query to explore your data and find your activation metric
```